### PR TITLE
`.com` to `.dev`

### PR DESCRIPTION
**Pull request by ghost**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### Description of the Change
Pulsar-edit.com to Pulsar-Edit.dev. This decision was made hours after PR #1 .

### Alternate Designs
N/A

### Benefits
N/A

### Possible Drawbacks
Some pages not removed?

### CODE REVIEW

**Comment by ghost** [comment]
@kaosine If able to review then you can.

File: lib/atom-io-client.coffee [hunk 1]
```diff
@@ -8,7 +8,7 @@ request = require 'request'
 module.exports =
 class AtomIoClient
   constructor: (@packageManager, @baseURL) ->
-    @baseURL ?= 'https://pulsar-edit.com/api/'
+    @baseURL ?= 'https://pulsar-edit.dev/api/'
```
Comment (kaosine) [inline lib/atom-io-client.coffee]:
```suggestion
    @baseURL ?= 'https://api.pulsar-edit.dev/'
```

**Review by kaosine — COMMENTED** [review]
Not sure if I got this right, @confused-Techie may have to confirm. Esp since I don't have packages pages yet since I'm not sure the functionality is there yet with the db to have it generate pages or some kinda parsable format for us.

File: lib/atom-io-client.coffee [hunk 2]
```diff
@@ -8,7 +8,7 @@ request = require 'request'
 module.exports =
 class AtomIoClient
   constructor: (@packageManager, @baseURL) ->
-    @baseURL ?= 'https://pulsar-edit.com/api/'
+    @baseURL ?= 'https://api.pulsar-edit.dev/'
```
Comment (confused-Techie) [inline lib/atom-io-client.coffee]:
```suggestion
    @baseURL ?= 'https://api.pulsar-edit.dev/api/'
```

**Review by confused-Techie — CHANGES_REQUESTED** [review]
Alright, so this all looks good. Added one change that needs to be made.

Otherwise looks great.

This isn't against anything in the PR, just making a mental note to myself and others.

This also show quite a bit of the Package Web GUI, which doesn't have any documentation.

This PR also create the web UI on the Domain itself, rather than the subdomain. We can always change this later if needed, but to put out some quick documentation on the layout of the Web GUI.

- URL/packages
- URL/themes
- URL/packageType/packageName
- URL/users/userName

**Review by confused-Techie — APPROVED** [review]
This looks great to me!

**Pull request merged**